### ISSUE TITLE
EVPN-VXLAN support

### ISSUE THREAD

**Issue by vkratsberg** [OpenBMP/obmp-psql#6]
Hello, 

Are there any plans to support EVPN-VXLAN routes ?

<img width="1094" alt="Screen Shot 2022-03-11 at 9 55 45 PM" src="https://user-images.githubusercontent.com/11032416/158001082-f73638cb-3702-437f-a7e0-d7ff590f1ab8.png">

**Comment by TimEvens** [comment]
Yes, I might be able to add EVPN next week. It depends on a LAB for testing and if time permits.  L3VPN is finally pretty solid now with the latest changes (version 2.1.1).

**Comment by xenque2** [comment]
Hello!

Do you have some plans to finish this?
